src/Playwright/API/Generated/.editorconfig

```diff
@@ -15,4 +15,8 @@ dotnet_diagnostic.SA1606.severity = none
 dotnet_diagnostic.SA1614.severity = none
 dotnet_diagnostic.SA1615.severity = none
 dotnet_diagnostic.SA1618.severity = none
-dotnet_diagnostic.SA1629.severity = none
\ No newline at end of file
+dotnet_diagnostic.SA1629.severity = none
+dotnet_diagnostic.CA1041.severity = none # [Obsolete] can exist without message
+dotnet_diagnostic.CS0612.severity = none # We can use [Obsolete] in generated code
+dotnet_diagnostic.SA1300.severity = none # Sometimes, we want to use _ in the name
```

Comment:
when do we want to use underscore?

Reply:
An API suggestion from upstream would be prefixed `_` and it's actually _acceptable_ IMO. It's a useless rule for generated files.

Reply:
We genuinely don't want underscore APIs in language bindings. If they are generated on your side, we need to fix it in the upstream API (we had it a few times and removed them then, e.g. response interception or fetch). So I'd remove that message pragma.